frontend/src/components/FollowersList.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import '../styles/App.css';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+import faExternalLinkAlt from '@fortawesome/fontawesome-free-solid/faExternalLinkAlt';
+import 'animate.css';
+import { withRouter } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+
+class FollowersList extends Component {
+  render() {
+    let { followers } = this.props;
+    return (
+      <div className="row">
+        {followers.map((followers, index) => {
+          let style = {
+            animationDelay: (index % 3) / 10 + 's'
+          };
+          return (
+            <div
+              key={index}
+              className="follower col-lg-3 col-md-5 m-2 animated fadeInUp"
```

Comment:
I don't think ```follower``` is a css class. If you were looking at some of the card code. ```card``` is Actually a bootstrap class.
https://getbootstrap.com/docs/4.0/components/card/

You can also use reactstrap to use Card as an element instead of a class
https://reactstrap.github.io/components/card/